pkg/plugins/common/kustomize/v2/scaffolds/internal/templates/config/manager/config.go

```diff
@@ -30,7 +30,8 @@ type Config struct {
 	machinery.ProjectNameMixin
 
 	// Image is controller manager image name
-	Image string
+	Image           string
+	ImagePullPolicy string
```

Comment:
```suggestion

	// ImagePullPolicy is ..
	ImagePullPolicy string
```

Could we ensure that we doc this one?